### PR TITLE
#130add_logo_link_bling

### PR DESCRIPTION
**Pull request by rbertolette**
For issue #130 Could possibly be improved by passing logo & white_logo vars to js, but I thought since white_logo is hardcoded in template.php it wasn't worth it.
Signed-off-by: Reed Bertolette reed@bertopro.com

### CODE REVIEW

File: sites/all/themes/playbox_theme/css/logo-bling.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	position: relative;
+	cursor: default;
+}
+
+.ch-img-1 { 
+	background: transparent url(../../playbox_theme/logo-white.svg) no-repeat center;
+}
+
+.ch-info {
+	position: absolute;
```
Comment (RobLoach) [inline sites/all/themes/playbox_theme/css/logo-bling.css]: May want to apply [Drupal CSS styling](https://www.drupal.org/coding-standards/css).

File: sites/all/themes/playbox_theme/js/custom.js [hunk 2]
```diff
@@ -107,6 +107,19 @@
 	Disco Party Time!
 	==============================================*/
 	$( document ).ready(function(){
+    
+    // added for logo_bling
+    $('#logo-hover').click(function(){
+      $(this).hide( "slow", function() {
+        if (window.location.pathname != '/'){
+          window.location.href = "/";
```
Comment (RobLoach) [inline sites/all/themes/playbox_theme/js/custom.js]: May be able to use `Drupal.basePath` rather then `"/"`.

**Comment by RobLoach** [comment]
Nice work, @rbertolette. Looks pretty neat when in effect. Added a few notes on where it could improve.

> I thought since white_logo is hardcoded in template.php it wasn't worth it.

Could make it not hard-coded if that would make your life easier :wink: .

**Comment by rbertolette** [comment]
@RobLoach just pushed changes you'd recommended for #130

**Pull request merged**